phpseclib/Crypt/TripleDES.php

```diff
@@ -267,6 +265,32 @@ function setIV($iv)
     }
 
     /**
+     * Sets the key length.
+     *
+     * Valid key lengths are 64, 128 and 192
+     *
+     * @see Crypt_Rijndael:setKeyLength()
```

Comment:
Should this be `@see Rijndael::setKeyLength()` or `@see \phpseclib\Crypt\Rijndael::setKeyLength()` since this comment isn't actually in Rijndael.php but rather TripleDES.php?

By that same token if, in TripleDES.php, there were a `@see Base::funcName()` should that be explicitly namespace'd out even if `@see Tripledes::funcName()` wouldn't be since Base is technically a different class? Or for phpdoc purposes does it not matter?